hermes/probe/delete/delete.go

```diff
@@ -13,9 +13,9 @@
 // limitations under the License.
 //
 // Delete_file implements the probe operation for deleting a file in a
-// storage system.
+// storage systemetrics.
```

Comment:
Typo.

Reply:
Refactoring... Thanks for spotting this, I've amended it.